controller/model/posture_check_type_model.go

```diff
@@ -0,0 +1,82 @@
+/*
+	Copyright NetFoundry, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package model
+
+import (
+	"github.com/openziti/edge/controller/persistence"
+	"github.com/openziti/fabric/controller/models"
+	"github.com/openziti/foundation/storage/boltz"
+	"github.com/pkg/errors"
+	"go.etcd.io/bbolt"
+	"reflect"
+)
+
+type PostureCheckType struct {
+	models.BaseEntity
+	Name             string `json:"name"`
+	OperatingSystems []OperatingSystem
```

Comment:
since you lowercased Name -> name... did you intend to NOT lowercase OperatingSystems?

Reply:
That is copypasta from a by-gone era. Removed.